scripts/tracecontext-integration-test.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/sh
-set -e 
+set -e
 # hard-coding the git tag to ensure stable builds.
-TRACECONTEXT_GIT_TAG="98f210efd89c63593dce90e2bae0a1bdcb986f51"
+TRACECONTEXT_GIT_TAG="e99a85d12df8c25c51ec1d5d234f20e918265b75"
```

Comment:
I think won't be so to easy to point to the lastest :) There's a failing test `test_tracestate_duplicated_keys`.

In https://github.com/w3c/trace-context/pull/477 
> The test_tracestate_duplicated_keys was broken, it demanded to drop
the whole tracestate when a duplicate key was found. This does not match
what the specification mandates in that situation. Instead,
implementations MAY remove the duplicates from tracestate or even leave
them in as-is.